tests/test_script_production.py

```diff
@@ -1248,7 +1248,9 @@ def test_env_emits_warning(self):
                     )
                 )
```

Comment:
would we...do the replace("\\", ..) on the path part here and that's it?  I'm not understanding this escaping should we just make it an r"" string?

Reply:
the problem is that `assertions.expect_warnings(mgs)` transforms `mgs` (typo.. 😞) to a regexp, so that causes problem.

So we have something like this

```py
msg = '''someting something
C:\\foo\\bar\\path
foo bar
'''
# in expect_warnings
re.compile(msg)
```

this does not work, since we an error like `error: bad escape \p at position 29 (line 2, column 11)`

Reply:
I have not checked if we can disable the regexp parse in `expect_warnings`. Maybe that's the best solution

Reply:
oh it's a reg.   OK...maybe add an expect_warnings_plain() function then.   this test is manufacturing the assertions from the data which is kind of backwards.